assets/src/edit-story/components/library/panes/media/mediaElement.js

```diff
@@ -173,7 +176,24 @@ const MediaElement = ({
 
   const onClick = () => onInsert(resource, width, height);
 
+  /// Callback for when the More dropdown menu has been opened / closed.
+  const menuCallback = (isMenuOpen) => {
+    if (isMenuOpen && type === 'video' && mediaElement.current) {
+      // If it's a video, pause the preview while the dropdown menu is open.
+      mediaElement.current.pause();
+      mediaElement.current.currentTime = 0;
+    } else if (!isMenuOpen) {
+      setShowMoreIcon(false);
+    }
+  };
+
   if (type === 'image') {
+    // TODO: Set to `true` when Edit and Delete have been fully implemented and merged.
+    // Currently we don't have feature flags (#1464) so this is a simple way to keep this feature
+    // hidden until fully implemented.
+    const onPointerEnterImage = () => setShowMoreIcon(false);
```

Comment:
Instead of having `onPointerEnterImage` and `onPointerEnterVideo`, wouldn't it be easier to have 1 `pointerEnter`?

E.g.

```js
const onPointerEnter = useCallback(() => {
  // TODO: Set to `true` when Edit and Delete have been fully implemented and merged.
  // Currently we don't have feature flags (#1464) so this is a simple way to keep this feature
  // hidden until fully implemented.
  setShowMoreIcon(true);

  if (type === 'video') {
    setShowVideoDetail(false);
    if (mediaElement.current) {
      mediaElement.current.play();
    }
  }
}, [type]);

const onPointerLeave = useCallback(() => {
  setShowMoreIcon(false);

  if (type === 'video') {
    setShowVideoDetail(true);
    if (mediaElement.current) {
      mediaElement.current.pause();
      mediaElement.current.currentTime = 0;
    }
  }
}, [type]);
```

Reply:
btw, the whole `currentTime = 0` thing seems super confusing. We might want to remove that because of the jumps/flickers this causes.

Reply:
Changed this part a bit to use `useEffect` instead, WDYT? 

And agreed, removed `currentTime = 0` for `pause` - but kept for `stop`.